src/daos/rinkeby.json

```diff
@@ -10,21 +10,42 @@
       "logo": "https://raw.githubusercontent.com/1Hive/dao-list/master/src/assets/rinkeby/0x7777cd7c9c6d3537244871ac8e73b3cb9710d45a/wrappedToken.svg"
     },
     "logo": "https://raw.githubusercontent.com/1Hive/dao-list/master/src/assets/rinkeby/0x7777cd7c9c6d3537244871ac8e73b3cb9710d45a/logo.svg",
+    "full_logo": "https://raw.githubusercontent.com/1Hive/dao-list/master/src/assets/rinkeby/0x7777cd7c9c6d3537244871ac8e73b3cb9710d45a/full_logo.svg",
```

Comment:
What do you think on renaming to `logo_type`?

Reply:
The header is using two different assets for the logo, one for small resolutions and another for large ones. 
In our case the bee an bee+1hive. 
This is why I defined a full_logo, that will be the icon+text. Does it make sense?

Reply:
Agree with the name, I'll update it